tests/apis/platformvm/addproposaltx.test.ts

```diff
@@ -0,0 +1,151 @@
+import { Buffer } from "buffer/"
+import { PlatformVMConstants } from "src/apis/platformvm"
+import { AddProposalTx } from "src/apis/platformvm/addproposaltx"
+import { BinTools } from "../../../src"
+import { UnsignedTx } from "../../../src/apis/platformvm"
+import { Serialization } from "../../../src/utils"
+describe("AddMemberProposalTx", (): void => {
+  const bintools = BinTools.getInstance()
+  const serialization = Serialization.getInstance()
+  const addProposalTxHex: string =
+    "000000002010000003ea00000000000000000000000000000000000000000000000000000000000000000000000159eb48b8b3a928ca9d6b90a0f3492ab47ebf06e9edc553cfb6bcd2d3f38e319a00000007000002b7b120fd800000000000000000000000010000000146a9c04f4bf783aa69daabd519dcf36978168b6600000001ee8f6da20ad416d4e6ea33a4575a2124d2d7a1d67ced49575e7b0264d4161b320000000059eb48b8b3a928ca9d6b90a0f3492ab47ebf06e9edc553cfb6bcd2d3f38e319a00000005000002b7b1303fc00000000100000000000000000000000e3c703e617364666173643c2f703e0000002a00000000201659f116d65aceba6d22cc557e8f3ee17e22c7757d0000000066526d700000000066a1877046a9c04f4bf783aa69daabd519dcf36978168b660000000a0000000100000000000000020000000900000001af93f89bcf175e1fe22429e3f8aa3d53088324f7bdd36d6b0f99f2dcd8216beb54842bbf35230ee9b3ff45d3fe9187feff3bb9dd5488976eeb4fc14d66104250000000000900000001af93f89bcf175e1fe22429e3f8aa3d53088324f7bdd36d6b0f99f2dcd8216beb54842bbf35230ee9b3ff45d3fe9187feff3bb9dd5488976eeb4fc14d6610425000a387e456"
+  const unsignedTx = new UnsignedTx()
+  unsignedTx.fromBuffer(Buffer.from(addProposalTxHex, "hex"))
+  const addProposalTx = unsignedTx.getTransaction() as AddProposalTx
+
+  test("getTypeName", async (): Promise<void> => {
+    const addProposalTxTypeName: string = addProposalTx.getTypeName()
+    expect(addProposalTxTypeName).toBe("AddProposalTx")
+  })
+
+  test("getTypeID", async (): Promise<void> => {
+    const addProposalTxTypeID: number = addProposalTx.getTypeID()
+    expect(addProposalTxTypeID).toBe(PlatformVMConstants.ADDPROPOSALTX)
+    console.log("addProposalTx decoded: ", addProposalTx)
+  })
+
```

Comment:
pleases add tests for `getProposalPayload` , `getProposalDescription` and `getProposerAuth`

Reply:
getProposerAuth test missing?